src/software/ai/play_selection_fsm.h

```diff
@@ -31,7 +42,9 @@ struct PlaySelectionFSM
      *
      * @param ai_config the default play config for this play fsm
      */
-    explicit PlaySelectionFSM(TbotsProto::AiConfig ai_config);
+    explicit PlaySelectionFSM(
+        const TbotsProto::AiConfig& ai_config,
+        std::shared_ptr<Strategy> strategy = std::make_shared<Strategy>());
```

Comment:
Is there a reason why `strategy` is given a default value here and in all the Play constructors? Kinda obfuscates that Strategy should be initialized in AI and passed in

Reply:
the reason was honestly just pure laziness on my part actually because I didn't want to go and update the test files

Reply:
Doesnt really matter, we can change it later